actors/multisig/tests/multisig_actor_test.rs

```diff
@@ -1,11 +1,16 @@
+use std::collections::HashMap;
```

Comment:
@Stebalien this is only in test code, but do you have something to say about the use of HashMap in this repo at all?

Reply:
Test code is fine.